Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server.py

```diff
@@ -1079,6 +1079,35 @@ def test_module(params: dict) -> str:
     return 'ok'
 
 
+def get_server_info_command(integration_context):
+    server_info = integration_context.get('server_info', None)
+
+    hr = '**In case the default/api_roots URL is incorrect, you can override it by setting' \
+         '"TAXII2 Service URL Address" field in the integration configuration**\n\n'
+    hr += tableToMarkdown('Server Info', server_info)
```

Comment:
move to note to the `metadata` argument in `tableToMarkdown` so it will be between the title and the table